Sources/Sentry/Public/SentryOptions.h

```diff
@@ -268,6 +268,12 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableSwizzling;
 
+@property (nonatomic, assign) NSInteger profilerSampleRateHz;
+
+@property (nonatomic, assign) BOOL measureProfilerCost;
+
+@property (nonatomic, assign) BOOL profilerEnabled;
```

Comment:
`m`: We have a naming pattern of starting with `enable`, see `enableSwizzling`, `enableNetworkBreadcrumbs`, `enableAutoSessionTracking`, `enableOutOfMemoryTracking`, `enableAutoPerformanceTracking`, ...
```suggestion
@property (nonatomic, assign) BOOL enableProfiler;
```